docs/modules/ROOT/pages/how-tos/upgrade-v1-v2.adoc

```diff
@@ -0,0 +1,56 @@
+= Upgrade `component-cert-manager` from `v1.0.0` to `v2.0.0`
+
+This upgrade upgrades the underlying `cert-manager` helm chart from `v0.15.1` to `v1.4.0`. There are some breaking changes in `cert-manager`. More information can be found https://github.com/jetstack/cert-manager/releases[here] and https://cert-manager.io/docs/installation/upgrading/[here].
```

Comment:
Please follow the style guide https://syn.tools/syn/references/style-guide.html:
1 line per sentence
```suggestion
This upgrade upgrades the underlying `cert-manager` helm chart from `v0.x` to `v1.x`. 
There are some breaking changes in `cert-manager`. 
More information can be found https://github.com/jetstack/cert-manager/releases[in changelog on GitHub] and in the https://cert-manager.io/docs/installation/upgrading/[upgrade documentation].
```

Reply:
ok thanks!